src/lesson1/HomeWorkApp.java

```diff
@@ -0,0 +1,67 @@
+package lesson1;
+
+public class HomeWorkApp {
+    public static void main(String[] args) {
+//      Для задачи 3
+        HomeWorkApp testObject = new HomeWorkApp();
+        int a = 1, b = 4;
+
+//        Задача 2
+        printThreeWords();
+
+//        Задача 3
+        testObject.checkSumSign(a, b);
+        testObject.checkSumSign(-3, -2);
+
+//        Задача 4
+        printColor(-8);
+
+//        Задача 5
+        compareNumbers(-4, -7);
+
+    }
+
+
+
+    public static void printThreeWords() {
+        String[] words = {"Orange", "Banana", "Apple"};
+
+        for (String word: words) {
+            System.out.println(word);
+        }
+    }
+
+
+
+    public void checkSumSign(int a, int b) {
+        int sum = a + b;
+
+        if (sum >= 0) {
+            System.out.println("Сумма положительная: " + sum);
+        } else {
+            System.out.println("Сумма отрицательная: " + sum);
+        }
+    }
+
+
+
+    public static void printColor(int value) {
+        if (value <= 0) {
+            System.out.println("Красный");
+        } else if (value > 0 && value <= 100) {
```

Comment:
```suggestion
        } else if ( value <= 100) {
```
поскольку первое условие автоматически выполняется из-за первой ветки оператора if